src/main/kotlin/com/progreizh/animaliste/dtos/PictureStorageDto.kt

```diff
@@ -0,0 +1,8 @@
+package com.progreizh.animaliste.dtos
+
+import org.springframework.web.multipart.MultipartFile
+
+data class PictureStorageDto(
```

Comment:
```suggestion
data class PictureDto(
```
Le nom que je préconise. En effet, le dto n'est pas un storage